src/woo_search/index/tasks/publications.py

```diff
@@ -0,0 +1,8 @@
+from woo_search.celery import app
+from woo_search.index.documents import Document
+
+
+@app.task()
+def index_document(data: dict):
+    Document.init()
+    Document(meta=data, **data).save()
```

Comment:
meta should probably only include the UUID (and perhaps the API resource URL back to publicatiebank?)